Update evernote extension

magnusmoaner opened the pull request:
## Description

<!-- A summary of your change. If you add a new extension or command, explain what it does. -->

## Screencast

<!-- If you add a new extension or command, include a screencast (or screenshot for straightforward changes). A good screencast will make the review much faster - especially if your extension requires registration in other services.  -->

## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

pernielsentikaer commented:
Hi @magnusmoaner👋 

Thanks for your contribution 🔥

Could you look into suggestions and add a CHANGELOG entry?

@artpi do you want to check this?

magnusmoaner commented:
> Hi @magnusmoaner👋
> 
> Thanks for your contribution 🔥
> 
> Could you look into suggestions and add a CHANGELOG entry?
> 
> @artpi do you want to check this?

Yes. Done.
I've updated the README, CHANGELOG, fixed a bit of error handling and removed console.log lines.

pernielsentikaer commented:
@artpi are you around?

magnusmoaner commented:
@artpi Can you please take a look at my changes?

pernielsentikaer reviewed (approved):
Hi 👋 

Looks good to me, approved 🔥
